### DISCUSSION TITLE
Unmap Alt-j key not work

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#1994] [Q&A] [answered]
Seems in lazyvim <kbd>A-j</kbd> make the current line move down. 
But I don't want it. I try to unmap it in my keymap file but not work. 
How to fix it?

```lua
local unmap = vim.api.nvim_del_keymap
unmap("n", "<A-j>")
unmap("x", "<A-j>")
```
And one more thing if I have to change the lazyvim built-in config to make it work. 
What is the nice way to make the change save to make own config file?
Since I just upload my config folder, anything else will missed outside.

--- 
# Update
Oh weird, I just `source` the keymap file. It warn me not such map. Means the <A-j> map may not in the nvim setting. 
What reason may cause this issue, I kill the keymap tool autokey in my pc, but not work either

**Comment by dpetka2001** [comment, marked as answer]
```lua
for _, mode in pairs({ "i", "v", "n" }) do
  vim.keymap.del(mode, "<A-j>")
end
```
Something like that should work in your `/lua/config/keymaps.lua` file.

**Comment by HermesSantos** [comment]
What plugin is doing this 'move' thing? idk why but when i press the esc button it is moving the current line up and down as well, thats why i want to disabled it.

**Reply by dpetka2001** [reply]
If you're inside tmux that's a Neovim-tmux problem. No plugin is doing that. See tmux [thread](https://github.com/tmux/tmux/issues/131#issuecomment-145853211) and Neovim [faq](https://neovim.io/doc/user/faq.html#faq) (search for `ESC IN TMUX OR GNU SCREEN IS DELAYED`).

**Reply by HermesSantos** [reply]
Im actually using tmux. Thank you

**Reply by dpetka2001** [reply]
Try to set `escape-time` to a very low value in your `tmux.conf` that it doesn't bother you. Recommended is 10-20, but I've seen ppl with 0 or 5.
